### ISSUE TITLE
Default start menu shortcuts target to invalid path

### ISSUE THREAD

**Issue by k0lourful** [Defaultplayer001/Deus-Ex-Universe-Community-Update-#1]
Target = "D:\Games\SteamLibrary\steamapps\common\Deus Ex\DeusExCommunityUpdate\System\DeusExCommunityUpdateint.exe"

There is no such file in this directory. Perhaps it's supposed to be "DeusEx.exe"?

**Comment by Defaultplayer001** [comment]
TLDR: "I'd recommend trying to reinstall!"

Sorry for the late reply! 

Still technically new to github, though I've been using it for a while now;.

I assumed I'd get an email about an issue or some kinda notice, but I don't seem to have?

Anyways, if you selected English during install there should be a "DeusExCommunityUpdateint.exe" file present in that location. (Or whatever location you selected to install to)

If something went wrong during install, the language files may not have been copied over. It's done as one of the very last steps.

I'd recommend trying to reinstall! 

Let me know about any other issues, and just so you know the Discord server is usually the fastest way to get help!

https://discord.gg/jCFJ3A6
